automation/common/deploy-kubevirt-and-cdi.sh

```diff
@@ -51,11 +50,11 @@ metadata:
   name: $CDI_NAMESPACE
 EOF
 
-LATEST_CDI=$(curl -L https://api.github.com/repos/kubevirt/containerized-data-importer/releases | \
-             jq '.[] | select(.prerelease==false) | .tag_name' | sort -V | tail -n1 | tr -d '"')
+# The version is fixed for release branch, because newer CDI versions may be incompatible
+CDI_VERSION="v1.50.0"
```

Comment:
Maybe add the same logic here as for Kubevirt? Always getting the latest dot release of a minor version?

Reply:
done